locales/en_US.json

```diff
@@ -62,12 +62,12 @@
 				"answer": "No. Lots of Pretendo's features/services are developed independently (for example, Miiverse may be worked on by one developer while Accounts and Friends is being worked on by another) and therefore we cannot give an overall ETA for how long this will take."
 			},
 			{
-				"question": "Does Pretendo work on Cemu/emulators?",
-				"answer": "Pretendo supports any client that can interact with Nintendo Network. Currently the only emulator with this kind of functionality is Cemu. Cemu 2.0 officially supports Pretendo under your network account options in the emulator. For information on how to get started with Cemu, check out the <a href='https://pretendo.network/docs/install/cemu'>documentation</a>.<br>Citra does not support true online play and thus does not work with Pretendo, and does not show signs of supporting true online play at all. Mikage, a 3DS emulator for mobile devices, may provide support in the future though this is far from certain."
+				"question": "When will you add more games?",
+				"answer": "We work on new games once we feel that our backend libraries are ready to support it, and there is developer interest in maintaining it. A lot of our work goes into stabilising and completing our existing games - we want to get the best experience possible in those before we move on to new titles. Since new work comes up all the time, we cannot make any estimate of when that would be."
```

Comment:
This can VERY easily be read as "Pretendo only wants to support games they think are cool", due to the wording of "and there is developer interest in maintaining it",which is not true.

I understand that *some* people on the team only wish to work on games they personally find interesting, but that is NOT the stance of the project as a whole, and there are plenty others who work on all games.

Our goal is to support all games within our scope, not just those which have developer interest.

Reply:
"Developer time available" maybe?